modules/getting-started/ref-audit-log-fields.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// assembly-audit-log.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-audit-log-fields.adoc_{context}"]
+= Audit log fields
+
+{product-short} audit logs can include the following fields:
+
```

Comment:
More for clarification: are these fields in the order they appear, or in the order they need to be filled out? Or is it a random order?

Reply:
This is a read-only output. I think they are automatically filled out as part of the audit log generation.